ui/src/components/utils/StringUtils.js

```diff
@@ -27,19 +27,23 @@ class StringUtils {
             scopeStr += 'All';
         } else if (
             item['scopeonprem'] === 'true' ||
-            item['scopeaws'] === 'true'
+            item['scopeaws'] === 'true' ||
+            item['scopegcp'] === 'true'
         ) {
             if (item['scopeonprem'] === 'true') {
                 scopeStr += 'OnPrem ';
             }
             if (item['scopeaws'] === 'true') {
                 scopeStr += 'AWS ';
             }
+            if (item['scopegcp'] === 'true') {
+                scopeStr += 'GCP ';
+            }
         } else {
             // Backward compatability - if no scope, assume on-prem
             scopeStr += 'OnPrem';
         }
-        return scopeStr.split(' ').sort().join(' ');
+        return scopeStr;
```

Comment:
without the `scopeStr.split(' ').sort().join(' ');` the scope field in microsegmentation table might look out of order like the below example
```
rule1 scope = AWS OnPrem
rule2 scope = OnPrem AWS 
```

Reply:
wasn't intended, made a mistake while rebasing. Updated